test

```diff
@@ -19,6 +19,7 @@ this_path="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 : ${MR_IMAGE_TAG:="0.1.2"}
 : ${MR_NETWORK:="testapi_default"}
 : ${MR_PORT:=8273}                    # this is your host machine's port
+: ${MR_PRIVILEGED:=false}             # certain machines require privileged access
```

Comment:
I want to note, by default we should have it on false, but some machines might need to toggle this to true when running ./test